doc/modules/cross_validation.rst

```diff
@@ -498,15 +499,19 @@ Here is a usage example::
 
   >>> from sklearn.model_selection import ShuffleSplit
   >>> X = np.arange(10)
-  >>> ss = ShuffleSplit(n_splits=5, test_size=0.25,
-  ...     random_state=0)
+  >>> ss = ShuffleSplit(test_size=0.25, random_state=0)
```

Comment:
Don't change this? Default is 10 and isn't deprecated

Reply:
Thanks to catch those, you are right.